src/app/pages/send/send-crypto-asset-form/form/stx/stx-send-form-confirmation.tsx

```diff
@@ -1,28 +1,70 @@
 import { useLocation } from 'react-router-dom';
 
+import { Stack } from '@stacks/ui';
+import { SendCryptoAssetSelectors } from '@tests/selectors/send.selectors';
 import get from 'lodash.get';
 
-import { ConfirmationButton } from '../../components/confirmation/components/confirmation-button';
-import { ConfirmationInfoLabel } from '../../components/confirmation/components/confirmation-info-label';
-import { SendFormConfirmationLayout } from '../../components/confirmation/components/send-form-confirmation.layout';
+import { FormAddressDisplayer } from '@app/components/address-displayer/form-address-displayer';
+import {
+  InfoCard,
+  InfoCardAssetValue,
+  InfoCardRow,
+  InfoCardSeparator,
+} from '@app/components/info-card/info-card';
+import { InfoLabel } from '@app/components/info-label';
+import { PrimaryButton } from '@app/components/primary-button';
+
 import { useStacksBroadcastTransaction } from '../../family/stacks/hooks/use-stacks-broadcast-transaction';
-import { StxSendFormConfirmationDetails } from './stx-send-form-confirmation-details';
 
 export function StxSendFormConfirmation() {
   const location = useLocation();
-
   const tx = get(location.state, 'tx');
-
-  const { stacksDeserializedTransaction, stacksBroadcastTransaction } =
+  const { stacksDeserializedTransaction, stacksBroadcastTransaction, formReviewTxSummary } =
     useStacksBroadcastTransaction(tx);
 
+  const {
+    txValue,
+    txFiatValue,
+    recipient,
+    fee,
+    totalBalance,
+    sendingValue,
+    arrivesIn,
+    nonce,
+    memoDisplayText,
+    symbol,
+  } = formReviewTxSummary(stacksDeserializedTransaction);
+
   return (
-    <SendFormConfirmationLayout>
-      <StxSendFormConfirmationDetails unsignedTx={stacksDeserializedTransaction} />
-      <ConfirmationInfoLabel symbol="stx" />
-      <ConfirmationButton
+    <InfoCard pt="extra-loose" pb="extra-loose" px="extra-loose">
+      <InfoCardAssetValue
+        value={Number(txValue)}
+        fiatValue={txFiatValue}
+        symbol={symbol}
+      ></InfoCardAssetValue>
```

Comment:
```suggestion
      <InfoCardAssetValue
        value={Number(txValue)}
        fiatValue={txFiatValue}
        symbol={symbol}
      />
```

thought we had a lint rule to prevent this